README.md

```diff
@@ -1,3 +1,36 @@
+# Add CSS modules support
```

Comment:
Yes, of course. It's just for reviewers to checkout the new API quickly.
Seems you are satisfied with current API, I will move them to the doc.